tests/integration/test_api/test_config/test_max_upload_size/test_max_upload_size.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+from os.path import join, dirname, realpath
+from uuid import uuid4
+
+import pytest
+import requests
+
+from wazuh_testing.tools.configuration import check_apply_test, get_api_conf
+
+# Marks
+
+pytestmark = pytest.mark.server
+
+# Configurations
+
+test_data_path = join(dirname(realpath(__file__)), 'data')
+configurations_path = join(test_data_path, 'conf.yaml')
```

Comment:
Maybe `conf.yaml` is a very generic name. The standard is `wazuh_<test_name>` in this case it would be
```suggestion
configurations_path = join(test_data_path, 'wazuh_max_upload_size.yaml')
```